airgun/views/hostgroup.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Text
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import (
+    ActionsDropdown,
+    FilteredDropdown,
+    SatTable,
+)
+
+
+class HostgroupView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Host Groups']")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text("./a"),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class HostgroupEditView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    title = Text("//span[contains(text(),'Edit')]")
+    compute_resource = FilteredDropdown(id='hostgroup_compute_resource')
```

Comment:
I don't see compute resource for default host group:
![image](https://user-images.githubusercontent.com/10895065/47086247-a348a180-d221-11e8-81a5-042da6f8c810.png)

Do we have special conditions for that?

Reply:
Compute resource == Deploy on